corehq/apps/email/views.py

```diff
@@ -0,0 +1,56 @@
+from django.http import HttpResponseRedirect
+from django.shortcuts import render, redirect
+from django.urls import reverse
+from django.utils.decorators import method_decorator
+from django.utils.translation import gettext as _
+
+from corehq import toggles
+from corehq.apps.domain.decorators import domain_admin_required, login_and_domain_required
+from corehq.apps.domain.views import BaseDomainView
+from corehq.apps.email.forms import EmailSMTPSettingsForm
+from corehq.apps.email.models import EmailSettings
+
+
+@login_and_domain_required
+def default(request, domain):
+    from corehq.messaging.scheduling.views import MessagingDashboardView
+    return HttpResponseRedirect(reverse(MessagingDashboardView.urlname, args=[domain]))
+
+
+@method_decorator(domain_admin_required, name='dispatch')
+@method_decorator(toggles.CUSTOM_EMAIL_GATEWAY.required_decorator(), name='dispatch')
+class EmailSMTPSettingsView(BaseDomainView):
+    template_name = 'email/email_settings.html'
+    urlname = 'email_gateway_settings'
+    page_title = _('Email Settings')
+    section_name = _('Messaging')
```

Comment:
It looks like these are translated with `gettext`, but they are loaded at module import, before there's a request.  I believe this should be `gettext_lazy` (or possibly `gettext_noop` - we're inconsistent about that).  We have a [docs page on that](https://commcare-hq.readthedocs.io/translations.html) if you haven't seen it.

Reply:
Thanks, changed to use `gettext_lazy` as specified in the doc.